google-cloud-storage/acceptance/storage_helper.rb

```diff
@@ -89,16 +89,16 @@ def try_with_backoff msg = nil, limit: 10
       end
     end
 
-    def self.run_one_method klass, method_name, reporter
-      result = nil
-      reporter.prerecord klass, method_name
-      (1..3).each do |try|
-        result = Minitest.run_one_method(klass, method_name)
-        break if (result.passed? || result.skipped?)
-        puts "Retrying #{klass}##{method_name} (#{try})"
-      end
-      reporter.record result
-    end
+    # def self.run_one_method klass, method_name, reporter
```

Comment:
Probably want to uncomment this before merging.

Reply:
WIP